.github/workflows/e2e-k8s.yml

```diff
@@ -47,11 +47,19 @@ jobs:
     needs: paths-filter
     if: ${{ (needs.paths-filter.outputs.not-ignore == 'true') || (github.event_name == 'push') }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8', '11' ]
```

Comment:
Adding JDK 11 to this job might not have much effect, as the current Docker image only supports JDK 8. Unless we also build a Docker image for JDK 11. We welcome your contribution of a JDK 11 image.

Reply:
> Adding JDK 11 to this job might not have much effect, as the current Docker image only supports JDK 8. Unless we also build a Docker image for JDK 11. We welcome your contribution of a JDK 11 image.

This is as the same as e2e.

Reply:
It seems we directly use mvnw for building image, which may not support JDK11.
I found that there is docker CLI in below steps.
How about we use mvnw for building jar, use Dockerfile for building image?
Is there any policy in apache against base docker image?

Reply:
> It seems we directly use mvnw for building image, which may not support JDK11. I found that there is docker CLI in below steps. How about we use mvnw for building jar, use Dockerfile for building image? Is there any policy in apache against base docker image?

`mvnw` executes Docker CLI commands through a plugin.

There are two stages here. The host machine needs to be in a JDK11 environment when executing `mvnw` to build the jar file. When building the Docker image, the base Docker image also needs to be JDK11.

Reply:
make sense. I will diginto `mvnw` base docker image setup, then.